test/unit/spec/EnrollChoices_spec.js

```diff
@@ -432,6 +432,7 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, R
             expect(test.form.isFactorMinimized('SYMANTEC_VIP')).toBe(false);
             expect(test.form.isFactorMinimized('RSA_SECURID')).toBe(false);
             expect(test.form.isFactorMinimized('DUO')).toBe(false);
+            expect(test.form.isFactorMinimized('WINDOWS_HELLO')).toBe(false);
```

Comment:
I guess this might not work because we removed windows hello from the factorlist

Reply:
No, we didn't remove WINDOWS_HELLO from anywhere.